### DISCUSSION TITLE
Coding day

### DISCUSSION THREAD

**Discussion by SamTov** [SamTov/MDSuite#384] [Announcements]
Hello all, 

With the paper soon out and therefore a release on PyPi I was hoping we could arrange a coding day. I think we could theoretically get up to 7 or so devs to help out so can we throw some dates around for the next two weeks?

@Fratorhe @PythonFZ @christophlohrmann @MarcBrueck @tobiasmerkt

**Comment by christophlohrmann** [comment]
on the 7th is espresso coding day, otherwise I have time

**Comment by PythonFZ** [comment]
I would like to leave out Tuesdays and the 01.12 but otherwise I have time.

**Comment by tobiasmerkt** [comment]
Hi Sam,

besides few lectures, I’m mostly available Mondays – Wednesdays. Thursdays and Fridays are packed with lectures and tutorials.

Cheers,
Tobias

**Comment by MarcBrueck** [comment]
next week from the works for me

**Comment by SamTov** [comment]
How would we all feel about the 8th?

**Reply by PythonFZ** [reply]
Sounds good to me

**Reply by MarcBrueck** [reply]
Same

**Comment by tobiasmerkt** [comment]
The 8th sounds good to me, too

Am 01/12/2021 um 17:25 schrieb Fabian Zills ***@***.***>:

﻿

Sounds good to me
